enderio-machines/src/main/java/com/enderio/machines/common/blocks/obelisks/ObeliskBlockEntity.java

```diff
@@ -2,50 +2,57 @@
 
 import com.enderio.base.api.UseOnly;
 import com.enderio.base.api.capacitor.CapacitorScalable;
+import com.enderio.base.api.filter.EntityFilter;
 import com.enderio.base.api.io.IOMode;
 import com.enderio.base.api.io.energy.EnergyIOMode;
+import com.enderio.base.common.init.EIOCapabilities;
 import com.enderio.machines.common.MachineNBTKeys;
 import com.enderio.machines.common.attachment.ActionRange;
 import com.enderio.machines.common.attachment.RangedActor;
 import com.enderio.machines.common.blocks.base.blockentity.PoweredMachineBlockEntity;
 import com.enderio.machines.common.blocks.base.blockentity.flags.CapacitorSupport;
 import com.enderio.machines.common.blocks.base.inventory.SingleSlotAccess;
 import com.enderio.machines.common.blocks.base.state.MachineState;
+import com.enderio.machines.common.blocks.base.state.MachineStateType;
 import com.enderio.machines.common.init.MachineAttachments;
 import com.enderio.machines.common.init.MachineDataComponents;
 import com.enderio.machines.common.io.IOConfig;
+import com.enderio.machines.common.lang.MachineLang;
 import com.enderio.machines.common.obelisk.ObeliskAreaManager;
+import java.util.Objects;
+import javax.annotation.Nullable;
 import net.minecraft.client.multiplayer.ClientLevel;
 import net.minecraft.core.BlockPos;
 import net.minecraft.core.HolderLookup;
 import net.minecraft.core.component.DataComponentMap;
 import net.minecraft.nbt.CompoundTag;
 import net.minecraft.server.level.ServerLevel;
+import net.minecraft.util.Mth;
 import net.minecraft.world.level.Level;
 import net.minecraft.world.level.block.Block;
 import net.minecraft.world.level.block.entity.BlockEntityType;
 import net.minecraft.world.level.block.state.BlockState;
 import net.minecraft.world.phys.AABB;
 import net.neoforged.fml.LogicalSide;
 
-import javax.annotation.Nullable;
-import java.util.Objects;
-
 public abstract class ObeliskBlockEntity<T extends ObeliskBlockEntity<T>> extends PoweredMachineBlockEntity
         implements RangedActor {
 
     private @Nullable AABB aabb;
     public static SingleSlotAccess FILTER = new SingleSlotAccess();
 
-    private static final ActionRange DEFAULT_RANGE = new ActionRange(5, false);
+    private boolean requiresFilter;
```

Comment:
I think this should be `final`? But honestly better yet I think the requiresFilter method should just be abstract - this means later obelisks can change the value depending on state.